### PR TITLE
Generic sc and st fiber patch panels

### PR DESCRIPTION
**Pull request by jhofmueller**
Adding three generic fiber patch panels:

* 24 port st
* 16 port st
* 12 port sc duplex

All with one splice rear port.

### CODE REVIEW

**Comment by danner26** [comment]
I am by far not an expert on these, @DanSheps any comments?

**Comment by DanSheps** [comment]
TBH, I don't know if this makes sense.

If it is a panel, you generally wouldn't have a splice.  This seems to far out there to be "generic" and is likely tied to a specific product.

**Comment by danner26** [comment]
@DanSheps kind of this [this PR](https://github.com/netbox-community/devicetype-library/pull/1807) they raised?

**Comment by DanSheps** [comment]
I don't know, honestly it is my view that these should all have individual rear ports for each front port since it is an individual splice and an individual strand.

Now this is just my view, but I think it is the most technically correct view when modelling any kind of fibre plant.

**Comment by danner26** [comment]
@DanSheps care to re-review this?

File: device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml
Comment (DanSheps) [inline device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml:1]: This file still has a multiple position rear port.  Is this intended?

**Review by DanSheps — CHANGES_REQUESTED** [review]
Sorry, I thought I responded to this for some reason.

File: device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml
Comment (jhofmueller) [reply device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml:1]: No, this was not intended. Fixed it.

Comment (DanSheps) [reply device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml:1]: Mark it ready for review when you are ready.

**Review by DanSheps — APPROVED** [review]

File: device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml
Comment (jhofmueller) [reply device-types/Generic/SC-12-port-duplex-fiber-patch-panel-rear-splice.yaml:1]: How would I do that? It's not a draft anymore, issues have been resolved and approved by you. Please advise.

**Pull request merged**